handler.py

```diff
@@ -0,0 +1,94 @@
+
+import os
+import logging
+import json
+import urllib.parse
+
+from rdflib.plugins.sparql.parser import parseUpdate
+from rdflib.plugins.sparql.algebra import translateUpdate
+
+from sns_client import SnsClient
+from neptune_client import NeptuneClient
+
+
+def main(event, _):
+    # Setup the logger at the INFO level while we continue to profile
+    logger = logging.getLogger()
+    logger.setLevel(logging.INFO)
+
+    rialto_sparql_endpoint = os.getenv('RIALTO_SPARQL_ENDPOINT', "http://localhost:8080/bigdata/namespace/kb/sparql")
+    rialto_sns_endpoint = os.getenv('RIALTO_SNS_ENDPOINT', "http://localhost:4575")
+    rialto_topic_arn = os.getenv('RIALTO_TOPIC_ARN', "rialto")
+    aws_region = os.getenv('AWS_REGION', "us-west-2")
+
+    sns_client = SnsClient(rialto_sns_endpoint, rialto_topic_arn, aws_region)
+    neptune_client = NeptuneClient(rialto_sparql_endpoint)
+
+    response, status_code = neptune_client.post(event['body'])
+
+    if "update=" in event['body'] and status_code == 200:
+        entities = get_unique_subjects(
+                        get_entities(
+                            urllib.parse.unquote_plus(
+                                event['body']).replace('update=', '')))
+        message = "{'Action': 'touch', 'Entities': %s}" % entities
```

Comment:
In testing, the message produced isn't liked by rialto derivatives. However, this worked:
```
message = {
            "Action": "touch",
            "Entities": entities
        }
        sns_response = sns_client.publish(json.dumps(message))
```